src/dynamics/ads_player/ads_player.js

```diff
@@ -595,6 +596,7 @@ Scoped.define("module:Ads.Dynamics.Player", [
                     dyn = dyn || this.parent();
                     if (Types.is_undefined(dyn.activeElement))
                         throw Error("Wrong dynamics instance was provided to _hideContentPlayer");
+                    dyn.set("outstreamoptions.hideOnCompletion", true);
```

Comment:
```suggestion
```

Reply:
Regarding this line, I agree will be better to add it inside close function, where it was calling only(as A Single Point of Truth). But I though not call this.parent() inside close function, so will refactor this line.